src/components/components/HeroSection/HeroSection.scss

```diff
@@ -0,0 +1,57 @@
+.HeroSection {
+	padding-top: 5%;
+	max-width: 1260px;
+	width: 80%;
+	max-height: 80vh;
+	margin: 0 auto;
+	display: flex;
+	align-items: flex-start;
+	gap: 5%;
+
+	&-main {
+		align-self: center;
+
+		&__title {
+			margin: 0;
+			font-size: calc(36px + 20 * (100vw / 1280));
+			line-height: 133%;
+			color: #151f4c;
+
+			&--circled {
+				padding: 0 10px;
+				border: 3px solid #0066ff;
+				border-radius: 30px;
+			}
+		}
+		&__text {
+			font-size: calc(16px + 2 * (100vw / 1280));
+			color: #7d7cac;
+			line-height: 150%;
+		}
+
+		&__button {
+			padding: 15.5px 24px;
+			color: #ffffff;
+			background-color: #0066ff;
+			border: 2px solid #0066ff;
+			border-radius: 8px;
+		}
+	}
+
+	&__image {
+		background: top right no-repeat url(img/hero-image.svg);
+		background-size: contain;
+		height: 100vh;
+		width: 90%;
+	}
+}
+
+@media (max-width: 767px) {
+	.HeroSection {
+		&-main {
+			&__title {
+				font-size: calc(36px + (20 + 20 * 0.7) * ((100vw - 320px) / 1280));
+			}
+		}
+	}
+}
```

Comment:
again the spacing looks weird. Please format it

Reply:
Have we agreed on what would be our team's standard as for the indentation - 2 spaces or 4? currently, my auto-formatting is set to 4 spaces, I believe.

Reply:
I have changed the indentation to be 2 spaces by default.